modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,54 @@
+= Merge Tags plugin
```

Comment:
This page needs to mention this is only available for 6.2 onwards by including the `include::partial$misc/admon-requires-6.2v.adoc[]` admonition similar to the footnotes docs PR.

Reply:
`include::partial$misc/admon-requires-6.2v.adoc[]` added to `mergetags.adoc`.